docs/RELEASE_POLICY.md

```diff
@@ -90,8 +90,9 @@ v3.1 will be patched since were the last two Firecracker releases and less than
 
 | Release | Release Date | Latest Patch | Min. end of support | Official end of Support         |
 | ------: | -----------: | -----------: | ------------------: | :------------------------------ |
+|    v1.8 |   2024-07-10 |       v1.8.0 |          2025-01-10 | Supported                       |
 |    v1.7 |   2024-03-18 |       v1.7.0 |          2024-09-18 | Supported                       |
-|    v1.6 |   2023-12-20 |       v1.6.0 |          2024-06-20 | Supported                       |
+|    v1.6 |   2023-12-20 |       v1.6.0 |          2024-06-20 | 2024-06-20 (end of 6mo support) |
```

Comment:
this is due to v1.8, isn't it?

Reply:
It was `end of support` before the release happen. How do we even select the reason here?

Reply:
It had been still supported because the invariant was "we support at least 2 releases at any time". If we had released 1.8 before 1.6 min end of support, 1.6 would have been evicted due to "end of 6mo support", but we dragged it until 1.8 was released to keep the 2 releases (1.6 and 1.7), so the reason we drop 1.6 support now is the 1.8 release.

Reply:
ok